visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -642,6 +636,19 @@ private void sauceVisualCheckLocal(String snapshotName, CheckOptions options) {
                 "return { height: window.innerHeight, width: window.innerWidth, dpr: window.devicePixelRatio }");
     String deviceName = String.format("Desktop (%sx%s)", dims.get("width"), dims.get("height"));
 
+    WindowScroll scroll = getWindowScroll();
+    List<RegionIn> ignoreRegions = extractIgnoreList(options);
+
+    for (WebElement element : options.getIgnoreElements()) {
+      RegionIn ignoreRegion = VisualRegion.ignoreChangesFor(element).toRegionIn();
+      ignoreRegions.add(ignoreRegion);
+    }
+
+    for (RegionIn region : ignoreRegions) {
+      region.setX(region.getX() - scroll.x);
```

Comment:
This should be `region.getX() + scroll.x`, no? Though this may only apply when doing full page